src/fedml/server_app.py

```diff
@@ -6,7 +6,7 @@
 from flwr.server import ServerApp, ServerAppComponents, ServerConfig
 from flwr.server.strategy import FedAvg
 
-from fedml.task import Net, get_weights
+from task import Net, get_weights
```

Comment:
```suggestion
from fedml.task import Net, get_weights
```

Use the package namespace to import. Otherwise you have to do weird import shenanigans and it makes distributing the project a pain! :)